mev-rs/src/validator_registry.rs

```diff
@@ -125,7 +125,7 @@ impl ValidatorRegistry {
     pub fn validate_registrations(
         &self,
         registrations: &mut [SignedValidatorRegistration],
-        current_timestamp: u64,
+        current_timestamp: u128,
```

Comment:
we should actually leave this as `u64` and if the caller is providing `u128` lets do the conversion once at that call site